Governance/concepts/backup.md

```diff
@@ -24,27 +24,27 @@ Apart from data loss you might also be concerned about your data being unavailab
 The next thing to consider is how big your data is. Your home directory can easily be backed up to a portable hard drive or to Dropbox, this is not really an option when you have terabytes of model output. However, supercomputer providers use a tape data archive, known as MDSS at NCI, to archive large files. Most institutions also have their own archives for important data created by their researchers. 
 
 ### How often do you update your data?
-You should also think about how often the data changes, this will determine how often you will need to back it up. Codes change often and using a version control system is the only way to really keep track of all the versions. Similarly, you might want to automatically update your documents on a daily basis. Model output or recorded observations are unlikely to change, you might need to back them up only once. Post processed output will change more frequently while you are still conducting your analysis, it might be worth to have some level of automation and back up your files every few days.
+You should also think about how often the data changes, this will determine how often you will need to back it up. Codes change often and using a version control system is the only way to really keep track of all the versions. Similarly, you might want to automatically update your documents on a daily basis. Model output or recorded observations are unlikely to change, you might need to back them up only once. Post processed output will change more frequently while you are still conducting your analysis, it might be worth having some level of automation and back up your files every few days.
 
 ### Characteristic of different backup options
-Not all backup solutions provide the same protection. Creating a copy of a file in a separate directory on your computer provides a measure of protection against accidental deletion, however it could be lost if your hard drive fails. On the other end of the scale you could store copies of your data on archival tapes in different cities, so that it could still be recoverable in the case of natural disasters. Generally, the latter is costly for a large data set, the amount of protection needed should be balanced against the value of your data.
+Not all backup solutions provide the same protection. Creating a copy of a file in a separate directory on your computer provides a measure of protection against accidental deletion, however it could be lost if your hard drive fails. On the other end of the scale you could store copies of your data on archival tapes in different cities, so that it could still be recoverable in the case of natural disasters. Generally, the latter is costly for a large data set and the amount of protection needed should be balanced against the value of your data.
```

Comment:
I wonder if we should check the book for style consistency in "data set" vs "dataset"?

Reply:
maybe we can discuss this at the next meeting I'm sure I use dataset all the time

Reply:
There's probably a few others to consider like ACDG vs ACDguide, netCDF4 vs netCDF-4, url vs URL

Reply:
Personally I would use:
ACDG
 netCDF4 
URL/url I know URL is the correct form but sometimes having a lot of acronyms makes the text hard to read, and url is in my mind like a word in itself, I never think of what it stands for,
Similarly for me, dataset is better in this context because we're not really talking about a set of data, we're usually referring about a group of sets of data coupled with metadata etc. Somehow, dataset seems to capture that better.